jina/peapods/pods/__init__.py

```diff
@@ -113,7 +113,7 @@ def _parse_args(self, args: Namespace) -> Dict[str, Optional[Union[List[Namespac
             if getattr(args, 'uses_after', None):
                 self.is_tail_router = True
                 peas_args['tail'] = _copy_to_tail_args(args)
-            peas_args['peas'] = _set_peas_args(args, peas_args.get('head', None), peas_args.get('tail', None))
+            peas_args['peas'] = _set_peas_args(args, peas_args.get('head'), peas_args.get('tail'))
```

Comment:
why is this dangerous? because `_set_peas_args` do not handle properly `None` ?

Reply:
Yes. It is unnecessary to provide None as the default value when the key is not present as it implicitly returns None.